modules/developer-cli-odo-switching-between-ephemeral-and-persistent-storage.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// *cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/working-with-storage.adoc
+
+[id="switching-between-ephemeral-and-persistent-storage_{context}"]
+= Switching between ephemeral and persistent storage
+
+You can switch between ephemeral and persistent storage in your project by using the `odo preference` command. `odo preference` modifies the global preference in your cluster.
+
+When persistent storage is enabled, the cluster stores the information between the restarts. 
+
+When ephemeral storage is enabled, the cluster stores temporary information that does not need to persist between the restarts.
+
+Ephemeral storage is enabled by default. 
+
+.Procedure
+
+. See the preference currently set in your project:
++
+[source,terminal]
+----
+odo preference view
+----
++
+.Example output
++
+[source,terminal]
+----
+PARAMETER             CURRENT_VALUE
+UpdateNotification
+NamePrefix
+Timeout
+BuildTimeout
+PushTimeout
+Experimental
+PushTarget
+Ephemeral             true
+----
+
+. To unset the ephemeral storage and set the persistent storage:
++
+[source,terminal]
+----
+$ odo preference unset ephemeral
```

Comment:
Simply unsetting the value will still use Ephemeral storage because it considers default value for Ephemeral when none is provided. And default value is `true`.

To ensure that odo uses Persistent storage for components, user needs to set the value to false:
`odo preference set Ephemeral false`.

This comment might help simplify things: https://github.com/openshift/odo/pull/4360#issuecomment-762162203.